e2e/manifests/mainnet.toml

```diff
@@ -4,6 +4,9 @@ public_chains = ["ethereum","arbitrum_one","base","optimism"]
 multi_omni_evms = true
 prometheus   = true
 
+[keys.tester]
```

Comment:
don't think this is the way to go, it deplicates whatever is in the eoa package. Follow the same patterns as relayer and monitor keys.